tests/components/airvisual/test_config_flow.py

```diff
@@ -0,0 +1,87 @@
+"""Define tests for the AirVisual config flow."""
+from unittest.mock import patch
```

Comment:
We should import patch from asynctest. Then we can patch both normal functions and coroutine functions.

Reply:
https://github.com/home-assistant/core/pull/32624